docker-compose.yml

```diff
@@ -1,16 +1,20 @@
-version: '2'
+version: '3'
 
 services:
+
   web:
-    build: .
+    container_name: bookit_web
+    image: builditdigital/bookit-web:latest
     ports:
-     - "8080:80"
+     - "80:80"
+    links:
+      - "server:bookitapi"
     environment:
-    - API_BASE_URL=http://localhost:8888
+      - BOOKIT_API_BASE_URL=/api
+    command: /bin/sh -c "envsubst < /opt/app/config.js.template > /opt/app/config.js && nginx -g 'daemon off;'"
+
   server:
```

Comment:
Where is the code for managing the EC2 instance?  I could swear I saw some Ansible/Terraform code somewhere at some point, but haven't managed to find it again.